src/sections/aliases.tsx

```diff
@@ -0,0 +1,211 @@
+import { Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from "@mui/material";
+import Section from "../future-hopr-lib-components/Section";
+import { useAppDispatch, useAppSelector } from "../store";
+import { useEffect, useRef, useState } from "react";
+import { actionsAsync } from "../store/slices/node/actionsAsync";
+import { exportToCsv } from "../utils/helpers";
+
+function AliasesPage() {
+  const dispatch = useAppDispatch();
+  const aliases = useAppSelector(selector => selector.sdk.aliases)
+  const loginData = useAppSelector(selector => selector.auth.loginData)
+
+  useEffect(() => {
+    if (loginData.ip && loginData.apiKey) {
+      dispatch(actionsAsync.getAliasesThunk({ apiEndpoint: loginData.ip, apiToken: loginData.apiKey }))
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [loginData])
+
+  return (
+    <Section
+      className="Section--aliases"
+      id="Section--aliases"
+      yellow
+    >
+      <h2>Add new alias</h2>
+      <CreateAliasForm />
+      <h2>Aliases table</h2>
```

Comment:
in this line, please also add a refresh button, which will download aliases again